Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp

```diff
@@ -1343,6 +1343,20 @@ void GraphicsLayerCA::setContentsDisplayDelegate(RefPtr<GraphicsLayerContentsDis
     noteLayerPropertyChanged(ContentsPlatformLayerChanged);
 }
 
+PlatformLayerIdentifier GraphicsLayerCA::setContentsToAsyncDisplayDelegate(ContentsLayerPurpose purpose)
+{
+    if (!m_contentsLayer) {
+        m_contentsLayer = createPlatformCALayer(PlatformCALayer::LayerTypeContentsProvidedLayer, this);
+        m_contentsLayerPurpose = purpose;
+        m_contentsLayer->setBackingStoreAttached(false);
+        m_contentsLayer->setAcceleratesDrawing(true);
```

Comment:
Also weird to force this on.

Reply:
This was just cargo-culted from the what we do for normal delegated display, I can remove it.